changelog/unreleased/kong/propagation-module-rework.yml

```diff
@@ -0,0 +1,5 @@
+message: |
+  **OpenTelemetry, Zipkin**: the propagation module has been reworked, new
+  options allow better control over the configuration of tracing headers propagation.
+type: feature
+scope: Plugin
```

Comment:
Should we consider this a core change?

Reply:
I thought about it. We cannot use multiple scopes and this is both a core and plugin change. I decided for the latter because from a user's perspective plugins are what is really affected by this.

Reply:
we can, but in this case it would be the same change and only the "scope" would be different. I don't know if we should create multiple identical changelog files with different scopes (wouldn't that create redundancy in the rendered changelog output?)